cfg_mgmt/gcp.py

```diff
@@ -124,3 +124,19 @@ def revert():
         )
 
     return revert
+
+
+def delete_config_secret(
+    cfg_factory: model.ConfigFactory,
+    cfg_queue_entry: cmm.CfgQueueEntry,
+) -> bool:
+    cfg_element = cfg_factory.container_registry(cfg_queue_entry.target.name)
+    logger.info('deleting old gcr secret')
+    iam_client = ccc.gcp.create_iam_client(
+        cfg_element=cfg_element,
+    )
+    delete_service_account_key(
+        iam_client=iam_client,
+        service_account_key_name=cfg_queue_entry.secretId['gcp_secret_key'],
+    )
+    return True
```

Comment:
what should be the semenatics of returning `False`? I think we should rather rely on exceptions (as this code does).

-> omit return-value